Lib/graphlib.py

```diff
@@ -22,7 +22,8 @@ def __init__(self, node):
 
 
 class CycleError(ValueError):
-    """Subclass of ValueError raised by TopologicalSorterif cycles exist in the graph
+    """Subclass of ValueError raised by :meth:`TopologicalSorter.prepare` if cycles
```

Comment:
Sphinx style formatting is not necessarily used in docstrings. Other changes look good to me.